doc/cmd_yarpview.dox

```diff
@@ -49,6 +49,8 @@ restore only the original aspect ratio (Image->Original aspect ratio).
 
 From the menu "File" you can also save individual or sequences of frames.
 
+You can connect to remote image source via gui. by remote name is not provided a disonnection command (from the last remote connected via gui) will be called
```

Comment:
The period is missing at the end of the sentence. The phrase "by remote name is not provided a disonnection command (from the last remote connected via gui) will be called" is not really clear.